lib/livebook_web/live/session_live/k8s_runtime_component.ex

```diff
@@ -777,4 +781,25 @@ defmodule LivebookWeb.SessionLive.K8sRuntimeComponent do
       pod_template: socket.assigns.pod_template.template
     }
   end
+
+  defp kubectl_warning() do
+    if System.find_executable("kubectl") == nil do
+      warning = "Could not find kubectl in PATH. Make sure to install it and add it to PATH."
+
+      if Livebook.Config.app?() do
+        windows? = match?({:win32, _}, :os.type())
+
+        path =
+          if windows? do
+            "%USERPROFILE%\\.livebookdesktop.bat"
```

Comment:
Should we expand this and tell users exactly what it is? It should be the USERPROFILE env var. We could do:

```suggestion
            "#{System.get_env("USERPROFILE", "%USERPROFILE%")}\\.livebookdesktop.bat"
```

I also wonder if we should show the specific command. Ideally the message for desktop would be:

```
Could not find kubectl in PATH. Make sure to install it and add it to PATH.

For Livebook Desktop, this can be done by creating a file at c:\\foo\\bar\\.livebookdesktop.bat with:

    set "PATH=c:\\path\\to\\dir\\;%PATH%"
```

Although, I wonder if we should support `KUBECTL_PATH` and `FLYCTL_PATH` env vars as well. Just because it is easier than telling people to fiddle with PATH:

```
Could not find kubectl in PATH. Make sure to install it and add it to PATH.

For Livebook Desktop, this can be done by creating a file at c:\\foo\\bar\\.livebookdesktop.bat with:

    set "KUBECTL_PATH=c:\\path\\to\\dir\\kubectl"
```

Then the conditional is:

```
executable = System.get_env("KUBECTL_PATH") || System.find_executable("kubectl")

if executable && File.regular?(executable) do
```

Thoughts?

Reply:
I think separate envs for each CLI is too much. There are definitely other scenarios where users would want to set the PATH, so this is just an opportunity to teach them how :)

Reply:
Updated
<img width="873" alt="image" src="https://github.com/user-attachments/assets/9aad8572-713f-4394-9aff-3e88ebb6d7e5">